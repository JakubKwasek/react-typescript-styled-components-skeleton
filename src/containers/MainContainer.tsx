import * as React from "react";
import { SimpleWidget } from "../components/SimpleWidget";

export const MainContainer = (): JSX.Element => <SimpleWidget name={"User"} />;
