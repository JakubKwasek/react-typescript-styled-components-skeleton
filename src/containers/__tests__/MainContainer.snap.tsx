import * as React from "react";
import * as renderer from "react-test-renderer";
import { MainContainer } from "../MainContainer";

describe("MainContainer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<MainContainer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
