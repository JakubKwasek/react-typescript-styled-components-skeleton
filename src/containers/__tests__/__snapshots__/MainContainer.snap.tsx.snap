// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer renders correctly 1`] = `
<div
  className="sc-AxjAm kVvkxP"
>
  <h2>
    Hello there:
  </h2>
  <h3>
    User
  </h3>
</div>
`;
